x-pack/plugins/security_solution/public/detections/components/callouts/missing_privileges_callout/translations.tsx

```diff
@@ -75,7 +75,7 @@ export const missingPrivilegesCallOutBody = ({
             />
             <ul>
               {indexPrivileges.map(([index, missingPrivileges]) => (
-                <li key={index}>{missingIndexPrivileges(index, missingPrivileges)}</li>
+                <li key={index}>{missingPrivilegesMessage(index, missingPrivileges)}</li>
```

Comment:
Line 74 contains `Missing Elasticsearch index privileges:`. Is it ok to have `index` here?

Reply:
Yes, it's ok.

Index privileges can be applied both to indices and data streams, refer to this data stream [doc](https://www.elastic.co/guide/en/elasticsearch/reference/current/securing-aliases.html#data-stream-privileges)